src/Tribe/Views/V2/Widgets/Widget_Abstract.php

```diff
@@ -74,6 +74,22 @@ public function setup_view( $arguments ) {
 		$this->filter_enqueue_assets( $context, $view );
 	}
 
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$instance" missing

<sub>hash: 2babadbecbde94d293f0529df05a350f</sub>